docs/TESTS.md

```diff
@@ -45,6 +45,14 @@ Then to test:
 $ make test
 ```
 
+### Clean up
+
+To clean up after a build (for example, in case a sunsequent build encounters problems) you can tell make to delete all of the buid output, including the current executable:
```

Comment:
```suggestion
To clean up after a build (for example, in case a subsequent build encounters problems) you can tell make to delete all of the build output, including the current executable:
```